File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/Chat_RestoreParams.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class Chat_RestoreParams(
  val modelID: String? = null,
  val messages: List<SerializedChatMessage>,
  val chatID: String,
)

File: agent/bindings/csharp/ChatRestoreParams.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class ChatRestoreParams
  {
    [JsonProperty(PropertyName = "modelID")]
    public string ModelID { get; set; }
    [JsonProperty(PropertyName = "messages")]
    public SerializedChatMessage[] Messages { get; set; }
    [JsonProperty(PropertyName = "chatID")]
    public string ChatID { get; set; }
  }
}
